src/main/java/org/gelecekbilimde/scienceplatform/post/model/request/CategoryCreateRequest.java

```diff
@@ -9,10 +12,27 @@
 @Setter
 @Builder
 public class CategoryCreateRequest {
-	@NotBlank
+
+	@NotBlank(message = "Category name cannot be blank")
+	@Size(max = 36, message = "Category name must be less than 36 characters")
 	private String name;
+
+	@NotBlank(message = "Description cannot be blank")
+	@Size(max = 255, message = "Description must be less than 255 characters")
+	private String description;
+
+	@NotNull(message = "Order number cannot be null")
+	@PositiveOrZero(message = "Order number must be zero or a positive integer")
 	private Integer order;
+
+	@NotBlank(message = "Slug cannot be blank")
+	@Size(max = 36, message = "Slug must be less than 36 characters")
 	private String slug;
+
+	@Size(max = 36, message = "Icon must be less than 36 characters")
 	private String icon;
+
+	@PositiveOrZero(message = "Parent ID must be zero or a positive number")
 	private Long parentId;
```

Comment:
Özel bir mesaj yazmamaya çalışalımi buradaki mesajları bu şekilde özelleştirdiğimizde yazması zor olur ve uyuşmaz. Mümkün olduğunca spring'in kendi varsayılan mesajlarını kullanalım. 

Ek olarak parent'ın ID bilgisi muhtemelen pozitif olmak zorunda olur çünkü veri tabanında ardışık olarak belirlenerek ilerleniyor. Yani 0 diye bir ID olmayacağından buradaki anotasyonu aşağıdaki şekilde güncellemek faydalı olabilr.
```suggestion
	@Positive
	private Long parentId;
```